components/Common/MetaBar/index.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import { CodeBracketIcon } from '@heroicons/react/24/outline';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import Image from 'next/image';
+import { FormattedMessage } from 'react-intl';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import MetaBar from './index';
+
+type Story = StoryObj<typeof MetaBar>;
+type Meta = MetaObj<typeof MetaBar>;
+
+export const Default: Story = {
+  args: {
+    items: {
```

Comment:
Clean Code :tm:

Reply:
It got much better with the suggestion in the review, thank you!